src/index.js

```diff
@@ -58,6 +58,8 @@ export function parseValue(value) {
     return parseMap(value);
   } else if (value === '') {
     return '""'; // Return explicitly an empty string (Sass would otherwise throw an error as the variable is set to nothing)
+  } else if (_.isString(value) && value.includes(',')) {
+    return '"' + value + '"'; // Sass throws an error if a value contains a comma and is not treated as a string
```

Comment:
What about something like `hsl(0,0,0)`? Sass wouldn't throw, right?

Reply:
You're right... I haven't thought about this... this makes it more difficult

Reply:
I'm unsure if this is something people use, but it'd be a breaking change. Is there a list of `SASS` functions we could refer to?

Reply:
I don't think, there is one. We would have to create one manually, I guess.
Maybe instead of checking a list of functions, we could just check for `(what, ever)`.

Reply:
I think that sounds like a good compromise!

Reply:
I've implemented the discussed changes.